spec/system/articles/user_visits_an_article_spec.rb

```diff
@@ -79,7 +79,7 @@
         visit articles.first.path
 
         elements = page.all(:xpath, articles_selector)
-        paths = elements.map { |e| e[:href] }
+        paths = elements.pluck(:href)
```

Comment:
Enumerable in Rails have the `pluck` method: https://api.rubyonrails.org/classes/Enumerable.html#method-i-pluck